server/springboot/src/main/java/com/styra/tickethub_springboot/dao/model/TicketRepository.java

```diff
@@ -1,12 +1,43 @@
 package com.styra.tickethub_springboot.dao.model;
 
+import com.styra.tickethub_springboot.App;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.data.domain.Sort;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.jpa.repository.QueryRewriter;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
-import java.util.List;
-
 @Repository
-public interface TicketRepository extends JpaRepository<Ticket, Integer> {
-    List<Ticket> findByTenant(Tenant tenant);
```

Comment:
☝️ Before, we've let JPA figure out what SQL should be used just from the method signature.

👇 Now, we'll have to spell out the query ourselves.